zulipterminal/config/themes.py

```diff
@@ -71,6 +71,7 @@
     'area:error'      : 'standout',
     'area:user'       : 'standout',
     'search_error'    : 'standout',
+    'active_narrow'   : 'standout',
```

Comment:
This commit is fine now, but the commit title seems inaccurate now?

Note that Zulip (and ZT) aims to use a text style which essentially means saying what the commit does rather than what it did, so eg "Add styles..." here, which also saves a few characters here and there :)

Reply:
Oh, okay! I'll change it. What do you think will be a better commit title?

Reply:
It's just one style, right? ```active_narrow```?

Reply:
Changed the commit message.